### PR TITLE
Increase timeout in nginx config for admin paths so we can validate 

### PR DESCRIPTION
**Pull request by katydidnot**
generic relationships on LinkEvents

Bug: T370901

## Description
Describe your changes in detail following the [commit message guidelines](https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines)

- Increased proxy_connect_timeout, proxy_send_timeout, and proxy_read_timeout to 120 seconds for admin path. 
- Increased send_timeout and keepalive_timeout to 120 seconds for admin path.

## Rationale

So we can validate if the generic relationship migration is successful, and to allow the admin dashboard some extra time. 

## Phabricator Ticket
https://phabricator.wikimedia.org/T370901

## How Has This Been Tested?
Not tested since we only have one environment 

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by suecarmol** [comment]
I'm also getting migration errors in my own (less complex) PR. I wonder if we have to manually run migrations in Wikilink

**Comment by jsnshrmn** [comment]
Have we tried simply increasing the timeout value in django_wait_for_migrations.py?

File: django_wait_for_migrations.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def wait_for_migrations(args):
         sleep(1)
         wait += 1
 
-        if wait > 30:
+        if wait > 500:
```
Comment (suecarmol) [inline django_wait_for_migrations.py:47]: Let's increase it to 2000 to see if it finally runs.

Comment (suecarmol) [reply django_wait_for_migrations.py]: 🤔 didn't work either. I'm not sure if increasing it more will help

Comment (katydidnot) [reply django_wait_for_migrations.py]: Yeah, I'm not sure either. 🤔

Comment (katydidnot) [reply django_wait_for_migrations.py]: I'm going to try 4000, and then look at some other solution. Have you all ever ran into this issue before?

**Pull request closed without merging**